test/generator/stubuser_generator.cpp

```diff
@@ -60,8 +60,8 @@ class StubUser : public Halide::Generator<StubUser> {
     }
 
     void schedule() {
-        const bool vectorize = true;
-        stub.schedule({ vectorize, LoopLevel(calculated_output, Var("y")) });
+        stub.set_vectorize(true)
```

Comment:
In non-stub usage I was imagining the scheduleparams would be public members and we'd be saying gen_instance.vectorize.set(true). Is that also what you're currently thinking? If so it would seem like a good idea to make stubs syntactically identical.